app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :set_commentable, only: %i[create]
+
+  def create
+    @comment = @commentable.comments.new(report_params)
+    @comment.user_id = current_user.id
```

Comment:
オブジェクトをそのまま代入できます 🙆 それか `current_user` の `comments` アソシエーションから `new` して、そこで  or それから `commentable` を設定してもよいです。

```suggestion
    @comment.user = current_user
```